Makefile

```diff
@@ -1,9 +1,11 @@
 # uncomment the one that's your C++ compiler
-CXX	= c++
+#CXX	= c++
 #CXX	= g++
-CXXFLAGS = -g
+# for debug
+#CXXFLAGS += -g -Wall -O0
+CXXFLAGS += -Wall
 
-all:	iffdigest.o
+all:	sf2-split sf2-test
```

Comment:
Given that this project is an IFF structure parsing library, making the main target be a set of command-line utilities for handling a specific file format doesn't make much sense.